emcc.py

```diff
@@ -4199,14 +4200,14 @@ def generate_traditional_runtime_html(target, options, js_target, target_basenam
     script.src = None
     script.inline = js_contents
 
-  html_contents = do_replace(shell, '{{{ SCRIPT }}}', script.replacement())
-  html_contents = tools.line_endings.convert_line_endings(html_contents, '\n', options.output_eol)
+  shell = do_replace(shell, '{{{ SCRIPT }}}', script.replacement())
+  if '{{{ CSS }}}' in shell:
+    shell = do_replace(shell, '{{{ CSS }}}', utils.read_file(utils.path_from_root('src/shell.css')))
+  if '{{{ LOGO_SVG }}}' in shell:
+    shell = do_replace(shell, '{{{ LOGO_SVG }}}', utils.read_file(utils.path_from_root('src/logo.svg')))
```

Comment:
`{{{ SCRIPT }}}` has always been part of our "API" for the shell file. Are these two new additions meant to be used by users? If so, we should document them, but if not, then perhaps they could use internal names to avoid theoretical conflicts with user code. In particular `{{{ CSS }}}` is just a few characters, and the `{{{ }}}` pattern is not unique to us (I learned it from Django IIRC), so I worry.

Reply:
Yes, I agree.  Will switch to something like `SHELL_CSS`.

Having said that:

1. we already use `SCRIPT` which is also very common
2. Folks with large / complex / non-trivial html should probably not be using emscripten-generated html / `--shell-file`.

Reply:
Do the only docs we have righ now the `{{{ SHELL }}}` seems to be:

```
The path name to a skeleton HTML file used when generating HTML output. The shell file used needs to have this token inside it: ``{{{ SCRIPT }}}``.
```

I'd rather not document `--shell-file` any more than this.  I can't think of a time when an external user would want to use a custom shell but then also embed the emscripten logo or CSS into that file.

Reply:
Agreed, yeah.